kafka-consumer-application/kafka/build.gradle

```diff
@@ -29,10 +29,12 @@ repositories {
 apply plugin: "com.github.johnrengelman.shadow"
 
 dependencies {
-    implementation "org.slf4j:slf4j-simple:2.0.7"
-    implementation "org.apache.kafka:kafka-clients:3.4.0"
-    testImplementation "junit:junit:4.13.2"
-    testImplementation 'org.hamcrest:hamcrest:2.2'
+    implementation 'org.slf4j:slf4j-simple:2.0.7'
+    implementation 'ch.qos.logback:logback-classic:1.4.14'
```

Comment:
logback needed?

Reply:
I added it to try to quiet the kafka loggers, but I think I can get it from a transitive dep. Lemme try.

Reply:
done